yacc/main.c

```diff
@@ -101,6 +101,18 @@ char *nullable;
 
 void done(int k)
 {
+    if ((action_file && ferror(action_file)) ||
+        (entry_file && ferror(entry_file)) ||
```

Comment:
Naive question from someone unfamiliar with C: is it actually valid to check say `entry_file` here, when the variable is declared without any explicit initialization? (Are those file variables implicitly NULL-initialized?) I see that older code below also tests the variables this way, so this is not new in the change. Is the behavior well-defined if `onintr` is called before we try to open the files?

Reply:
Global variables are always initialized to 0.